### ISSUE TITLE
[Feature Request] Gated Delta Net (Norm) Unfused Gate

### ISSUE THREAD

**Issue by CoffeeVampir3** [fla-org/flash-linear-attention#596]
### Feature Request

Having a variant of gated delta net that is not fusing RMSNorm into the gate would be ideal to support more options for normalization. Specifically, Qwen3 Next is using a variant of RMSNorm that is Zero centered with weight decay.

### Motivation

Qwen3 Next Architecture using Gated Delta Net but a different RMSNorm variant:

<img width="2340" height="2428" alt="Image" src="https://github.com/user-attachments/assets/2e000ac3-d517-4d98-9c3b-c32425996cf7" />

https://huggingface.co/Qwen/Qwen3-Next-80B-A3B-Instruct

### Your Contribution

I can write the PR if this is something of value to the repo 👍

**Comment by zhiyuan1i** [comment]
I'll look into this requirement in my free time, and if you're willing, you're welcome to submit a PR and submit a test

**Comment by CoffeeVampir3** [comment]
Probably unsurprising, unfusing the gate gives really fairly substantial memory inflation. It looks like the more viable approach is to fuse the zero centering RMS norm into the existing setup which makes things awkward because the initialization is inverted (Zeroes instead of ones) but I think the core centering should be trivial in the backward. This might be getting into the territory of variant scattering in the code how do you feel about add it as a fused option?

```python
class ZeroCenteredRMSNorm(nn.Module):
    def __init__(self, dim: int, eps: float = 1e-6):
        super().__init__()
        self.eps = eps
        self.weight = nn.Parameter(torch.zeros(dim))

    def forward(self, x):
        norm = x * torch.rsqrt(x.pow(2).mean(-1, keepdim=True) + self.eps)
        output = norm * (1.0 + self.weight.float())
        return output - output.mean(-1, keepdim=True)
```
